### PR TITLE
Make segment title clickable and add segment definition to tooltip.

### PR DESCRIPTION
**Pull request by diosmosis**
Couple small changes after talking to @mattab

### CODE REVIEW

**Comment by tsteur** [comment]
Are there any important fixes in there? Otherwise rather move it to 4.0 maybe.

**Comment by diosmosis** [comment]
No important fixes, just seemed small enough for 3.13.1. Will move it.

File: plugins/CoreHome/angularjs/comparisons/comparisons.component.html [hunk 1]
```diff
@@ -4,8 +4,8 @@ <h3>{{ 'General_Comparisons'|translate }}</h3>
     <div class="comparison card" ng-repeat="comparison in $ctrl.comparisonsService.getSegmentComparisons() track by $index">
         <div class="comparison-type">{{ 'General_Segment'|translate }}</div>
 
-        <div class="title" title="{{ comparison.title }}">
-            {{ comparison.title }}
+        <div class="title" title="{{ comparison.title }}&lt;br/&gt;{{ comparison.params.segment }}">
```
Comment (mattab) [inline plugins/CoreHome/angularjs/comparisons/comparisons.component.html]: not sure, but does it maybe need some escaping for XSS, as a segment definition can contain any string?

Comment (diosmosis) [reply plugins/CoreHome/angularjs/comparisons/comparisons.component.html]: angularjs does this by default, so should be ok.

Comment (tsteur) [reply plugins/CoreHome/angularjs/comparisons/comparisons.component.html]: @diosmosis this is what it looks for me

image

with this segment

image
in the DB the segment is stored as `pageUrl=@b`

**Review by tsteur — CHANGES_REQUESTED** [review]
See comment, there might be encoding issues?

**Comment by sgiehl** [comment]
@diosmosis any update on this?

**Comment by tsteur** [comment]
@diosmosis be good to look into this when you can

**Comment by diosmosis** [comment]
@sgiehl / @tsteur pushed a fix

**Comment by tsteur** [comment]
@diosmosis seems some tests are failing now because the font changes slightly?

https://builds-artifacts.matomo.org/matomo-org/matomo/4.x-dev/39205/Comparison_dashboard_last_period.png

https://builds-artifacts.matomo.org/matomo-org/matomo/4.x-dev/39205/Comparison_visitors_overview.png

Wondering if it is because there's now no newline anymore and the browser renders it slightly differently? https://github.com/matomo-org/matomo/pull/15269/files#diff-35a447f52ea628304209b9dcf28812baR8 I suppose be fine to add this newline again?

**Comment by diosmosis** [comment]
@tsteur could re add it and check, or could just copy over the screenshots? I don't really have an opinion.

**Comment by tsteur** [comment]
Not really. I'll test and add the spaces. Maybe it fixes the screenshots then it would avoid conflicts with other PR that updates same screenshots.

**Comment by tsteur** [comment]
@diosmosis didn't seem to change anything. Seems to be maybe because of adding a title?

**Comment by diosmosis** [comment]
probably, maybe the rendering changes order or something and there's some tiny subtle change. can just update the screenshots?

**Comment by tsteur** [comment]
sounds good 👍

**Pull request merged**